client-common/src/main/java/com/cloudera/livy/client/common/Serializer.java

```diff
@@ -59,7 +59,9 @@ protected Kryo initialValue() {
   }
 
   public Object deserialize(ByteBuffer data) {
-    Input kryoIn = new Input(new ByteBufferInputStream(data));
```

Comment:
Could you remove the `ByteBufferInputStream` import since it's now unused? LGTM otherwise.